modules/nf-core/foldseek/easysearch/main.nf

```diff
@@ -0,0 +1,52 @@
+process FOLDSEEK_EASYSEARCH {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::foldseek=8.ef4e960"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/foldseek:8.ef4e960--pl5321hb365157_0':
+        'biocontainers/foldseek:8.ef4e960--pl5321hb365157_0' }"
+
+    input:
+    tuple val(meta)   , path(pdb)
+    tuple val(meta_db), path(db)
+
+    output:
+    tuple val(meta), path("${meta.id}.m8"), emit: aln
+    path "versions.yml"                   , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    foldseek \\
+        easy-search \\
+        ${pdb} \\
+        ${db}/${meta_db.id} \\
+        ${prefix}.m8 \\
+        tmp \\
```

Comment:
is `tmp` a parameter ?

Reply:
This is a name for a tmp folder that foldseek uses for intermediate calculations. Renamed it to tmpFolder as they do in their docs: https://github.com/steineggerlab/foldseek